merlin/models/tf/transforms/sequence.py

```diff
@@ -621,6 +621,41 @@ def from_config(cls, config):
         return cls(schema, target, **config)
 
 
+@tf.keras.utils.register_keras_serializable(package="merlin.models")
+class SequenceMaskLastInference(Block):
```

Comment:
Wondering if we can simplify a bit the API for the user by making `SequenceMaskLastInference` a default `pre` of `ReplaceMaskedEmbeddings`

Reply:
That's a good point. `ReplaceMaskedEmbeddings` currently inherits from the `Block` class, which doesn't have a `pre` argument. 

We also discussed the solution to instantiate `ReplaceMaskedEmbeddings` as a sub-class of TabularBlock so as to use the `pre` property. However, `TabularBlock` is not updated to use the recent logic of `call_quentially`/`to_call` introduced in MM combinators, outputs, and encoder blocks. Instead, it relies on `_tabular_call` method that does not forward the **kwargs argument to the `pre` layer, such as training/testing flags.  

So, in the meantime, I think we should expose the `SequenceMaskLastInference` to the user. I also added a description in the `ReplaceMaskedEmbeddings` docstrings to explain the importance of `SequenceMaskLastInference`.